package.json

```diff
@@ -27,12 +27,26 @@
     "url": "https://github.com/lob/hapi-bookshelf-total-count/issues"
   },
   "homepage": "https://github.com/lob/hapi-bookshelf-total-count#readme",
+  "peerDependencies": {
+    "bookshelf": "0.9.x",
+    "hapi": "11.x"
+  },
```

Comment:
gotcha on the hapi-qs

for bookshelf though, i think it'd make sense to say "this version" and up. while its true you havent tested it against the older versions, there's no reason to lock it down against newer versions. if something happens and it doesn't work in the future, someone will make an issue and we'll fix it. if nothing happens and it does work, with this restrictive version, we'd still have to go into this repo to allow the newer version. if we just say "this and up", then it will just work with no necessary intervention, right?